src/main/java/HW13/Customer.java

```diff
@@ -0,0 +1,18 @@
+package HW13;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Accessors(chain = true)
+public class Customer {
+    private int ID;
```

Comment:
Naming is not right!
https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names
Non-constant field names (static or otherwise) are written in lowerCamelCase.
These names are typically nouns or noun phrases. For example, computedValues or index.